README.md

```diff
@@ -8,7 +8,7 @@ Generic helpers for GraphQL API's.
    ```elixir
    def deps do
      [
-       {:cose_della_vita_ex, "~> 0.0.0+development"}
+       {:cose_della_vita_ex, "~> 0.0.3"}
```

Comment:
Dit hoeft niet, gebeurt automagisch in CI

Reply:
Aaah, my bad